sdv/lite/single_table.py

```diff
@@ -0,0 +1,197 @@
+
+        return sampled
+
+    def sample_remaining_columns(self, known_columns, max_tries_per_batch=100, batch_size=None,
+                                 randomize_samples=True, output_file_path=None):
+        """Sample rows from this table.
+
+        Args:
+            known_columns (pandas.DataFrame):
+                A pandas.DataFrame with the columns that are already known. The output
+                is a DataFrame such that each row in the output is sampled
+                conditionally on the corresponding row in the input.
+            max_tries_per_batch (int):
+                Number of times to try sampling discarded rows. Defaults to 100.
+            batch_size (int):
+                The batch size to use per attempt at sampling. Defaults to 10 times
+                the number of rows.
+            randomize_samples (bool):
+                Whether or not to use a fixed seed when sampling. Defaults
+                to True.
+            output_file_path (str or None):
+                The file to periodically write sampled rows to. Defaults to
+                a temporary file, if None.
+
+        Returns:
+            pandas.DataFrame:
+                Sampled data.
+        """
+        sampled = self._synthesizer.sample_remaining_columns(
+            known_columns, max_tries_per_batch, batch_size, randomize_samples, output_file_path)
+
+        return sampled
+
+    def save(self, path):
+        """Save this model instance to the given path using cloudpickle.
+
+        Args:
+            path (str):
+                Path where the SDV instance will be serialized.
+        """
+        self._package_versions = get_package_versions(getattr(self, '_synthesizer', None))
+
+        with open(path, 'wb') as output:
+            cloudpickle.dump(self, output)
+
+    @classmethod
+    def load(cls, path):
+        """Load a SingleTableSynthesizer instance from a given path.
+
+        Args:
+            path (str):
+                Path from which to load the instance.
+
+        Returns:
+            SingleTableSynthesizer:
+                The loaded synthesizer.
+        """
+        with open(path, 'rb') as f:
+            model = cloudpickle.load(f)
+            throw_version_mismatch_warning(getattr(model, '_package_versions', None))
+
+            return model
+
+    @classmethod
+    def list_available_presets(cls, out=sys.stdout):
+        """List the available presets and their descriptions."""
+        out.write(f'Available presets:\n{PRESETS}\n\n'
+                  'Supply the desired preset using the `name` parameter.\n\n'
+                  'Have any requests for custom presets? Contact the SDV team to learn '
+                  'more an SDV Premium license.\n')
+
+    def __repr__(self):
+        """Represent single table preset instance as text.
+
+        Returns:
+            str
+        """
+        return f'SingleTablePreset(name={self.name})'
```

Comment:
This misses the metadata and may end up confusing a bit. Usually if we override this like that we should be able to   `eval(repr(instance))` and get a new instance, in this case this is not possible because missing metadata (which makes no sense to be represented in here). However I think that a different approach may be desired. For example:
```
SingleTablePreset:
    preset name: 'FAST_ML'
    metadata: {
        ...
    }
```

Reply:
I think in this case the significance of printing the name is to show what type of preset was used. I think it would be unnecessary to print out the entire metadata. As you mentioned, the purpose is not to make it so that a new instance could be created by copying the text